rules/windows/file_event/sysmon_detect_powerup_dllhijacking.yml

```diff
@@ -0,0 +1,24 @@
+title: Powerup Write Hijack DLL detection
+id: 602a1f13-c640-4d73-b053-be9a2fa58b96
+status: experimental
+description: Powerup tool's Write Hijack DLL exploits DLL hijacking for privilege escalation. In it's default mode, it builds a self deleting .bat file which executes malicious command. The detection rule relies on creation of the malicious bat file (debug.bat by default). Reference: https://powersploit.readthedocs.io/en/latest/Privesc/Write-HijackDll/
+author: pbssubhash
+date: 2021/08/21
+modified: 2021/08/21
+tags:
```

Comment:
Hi no need for modified as it is a first release

Reply:
Noted.